packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnecoCoinjoinModal.tsx

```diff
@@ -45,8 +45,9 @@ const AgreeButton = styled(Button)`
 
 export const UnecoCoinjoinModal = () => {
     const account = useSelector(selectSelectedAccount);
+    const localCurrency = useSelector(state => state.wallet.settings.localCurrency);
```

Comment:
nit:
use `selectLocalCurrency` selector, you can refactor it in separate PR if you want for all occurences 😃

Reply:
Ok will do

Reply:
57f2822a8eb60dbacd2924812cfb0c0a06da3760 (will refactor other places after this gets merged)